### PR TITLE
Fix build test temp dir

### PR DESCRIPTION
**Pull request by kylegc**

### CODE REVIEW

File: functest/build_test.go [hunk 1]
```diff
@@ -134,10 +134,9 @@ func TestBuildLabels(t *testing.T) {
 	}
 
 	// create a temporary dir to create the project and run the test
-	projectDir, err := ioutil.TempDir("", "appsody-build-labels-test")
-	if err != nil {
-		t.Fatal(err)
-	}
+	projectDir := cmdtest.GetTempProjectDir(t)
+	defer os.RemoveAll(projectDir)
+	t.Log("Created project dir: " + projectDir)
```
Comment (EnriqueL8) [inline functest/build_test.go]: We are doing these operations just below.

**Comment by EnriqueL8** [comment]
Also there are 6 commits showing up, might have been that the branch wasn't originaly rebased on master.

File: functest/build_test.go [hunk 2]
```diff
@@ -134,10 +134,9 @@ func TestBuildLabels(t *testing.T) {
 	}
 
 	// create a temporary dir to create the project and run the test
-	projectDir, err := ioutil.TempDir("", "appsody-build-labels-test")
-	if err != nil {
-		t.Fatal(err)
-	}
+	projectDir := cmdtest.GetTempProjectDir(t)
```
Comment (EnriqueL8) [inline functest/build_test.go]: I was also thinking if you would want to name the TempDir for debugging purposes?

Comment (EnriqueL8) [reply functest/build_test.go]: Might also be nicer to return a `cleanup` function which would be equivalent to the `os.RemoveAll` passed to `defer`

File: functest/build_test.go (hunk 1 above)
Comment (kylegc) [reply functest/build_test.go]: removed the duplicates

File: functest/build_test.go (hunk 2 above)
Comment (kylegc) [reply functest/build_test.go]: I agree with these comments, however `GetTempProjectDir()` has been used all over in the tests for quite some time now, so I'd like to get this fix in and address these comments in a separate PR.

Comment (EnriqueL8) [reply functest/build_test.go]: Sounds good!

**Pull request merged**